<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtCombo</class>
 <widget class="QWidget" name="QtCombo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtCombo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>简单ComboBox</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="btnInitCity">
          <property name="text">
           <string>初始化列表</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btnClearCity">
          <property name="text">
           <string>清空列表</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="btnCityEditable">
          <property name="text">
           <string>可编辑</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QComboBox" name="comboCity">
          <item>
           <property name="text">
            <string>兰州</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>武威</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>张掖</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>敦煌</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>酒泉</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>内部值ComboBox</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="btnInitCityNum">
          <property name="text">
           <string>初始化城市/区号</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="comboCItyNum">
          <property name="placeholderText">
           <string>请选择一个城市</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>显示</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnClear">
        <property name="text">
         <string>清空文本</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="txtShow"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
